from error import Error

class UnexpectedGameCloseError(Error):
    pass
